src/grt/include/grt/GlobalRouter.h

```diff
@@ -142,13 +139,19 @@ class GlobalRouter
  public:
   GlobalRouter();
   ~GlobalRouter();
+
+#ifndef SWIG
```

Comment:
It is better to handle this in the .i if possible to keep swig concerns out of the c++